tensorrt_llm/_torch/custom_ops/cpp_custom_ops.py

```
@@ -238,6 +238,313 @@ def _(
+    def _check_turboquant4_tokens_per_block(cache: torch.Tensor,
+                                            tokens_per_block: int) -> None:
+        if isinstance(cache.shape[2],
+                      int) and cache.shape[2] != tokens_per_block:
+            raise RuntimeError(
+                "TurboQuant4 cache tokens_per_block mismatch.")
+
+    def _check_turboquant4_block_capacity(block_ids: torch.Tensor,
+                                          seq_len: int,
+                                          tokens_per_block: int) -> None:
+        if not isinstance(seq_len, int) or not isinstance(block_ids.shape[-1],
+                                                         int):
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)